Archives/VolFile.cpp

```diff
@@ -367,11 +370,12 @@ namespace Archives
 	}
 
 	// Writes a section tag to the open output file.
-	void VolFile::WriteTag(SeekableStreamWriter& volWriter, int length, const char *tagText)
+	void VolFile::WriteTag(StreamWriter& volWriter, int length, const char *tagText)
 	{
 		int buffer[2];
 
 		buffer[0] = *(int*)tagText;
+		// Use a bitmask to increase tag size to a multiple of 4 bytes
 		buffer[1] = length | 0x80000000;
```

Comment:
I've explained poorly and forgot to mention this part. This comment is wrong. Here a bit field is being used. The code is setting the highest most bit, which is used to indicate how the section should be padded. I believe it uses padding up to either a 2 or 4 byte boundary. We should refactor the code (in the future) to use structs with an embedded bit field. That would make this code much more clear, and make errors less likely.

For now, perhaps we can comment this as:
Set padding bit to indicate 4 byte padding